spec/fieldhand/record_spec.rb

```diff
@@ -35,6 +35,27 @@ module Fieldhand
       end
     end
 
+    describe '#dump' do
+      it 'returns the whole element even if there is no metadata' do
+        element = ::Ox.parse('<record/>')
+        record = described_class.new(element)
+
+        expect(record.dump).to eq("\n<record/>\n")
+      end
+
+      it 'returns the whole element with its metadata as a string' do
```

Comment:
It just means that we return the whole element as a string, any suggestions for a better method description?